client/client-multi/client-objective-c/Kaa/channel/impl/channels/DefaultOperationTcpChannel.m

```diff
@@ -380,9 +380,26 @@ - (void)scheduleOpenConnectionTaskWithRetryPeriod:(int64_t)retryPeriod {
 
 - (void)schedulePingTask {
```

Comment:
Since it's not have an internal delay, it doesn't schedule but execute ping request, so method should be rename to avoid further confusing.